### ISSUE TITLE
Update the CellProfiler pipeline for pixel classification based segmentation

### ISSUE THREAD

**Issue by nilseling** [BodenmillerGroup/steinbock#191]
The CellProfiler pipeline used in steinbock does not match the current pipeline in the ImcSegmentationPipeline.

**Comment by jwindhager** [comment]
Out of curiosity: in what way?

**Comment by nilseling** [comment]
So steinbock is using the old pipeline where the probability masks were first segmented and then the detected objects were downscaled. This leads to issues that cells are lost if they are smaller than 4 pixels or so. 

The pipeline in the newest version of the ImcSegmentationPipeline first downscales the pixel probabilities and then performs segmentation. This leads to continuous numbering of cell IDs.

**Comment by nilseling** [comment]
Hmm, I can't reproduce this issue anymore. For the test data the cell IDs seem to be continous with the current pipeline. But I remember that there was an issue in the past when reading in segmentation masks that did not contain continous cell IDs. In any case, it would be good to update the pipeline to what the ImcSegmentationPipeline is doing.

**Comment by nilseling** [comment]
No, the two pipelines are conceptually quite different. We can have a look at it together tomorrow
